examples/p5js/NeuralNetwork/NeuralNetwork_ImageClassifier_Video/sketch.js

```diff
@@ -26,73 +26,34 @@ function setup() {
   video = createCapture(VIDEO);
   video.size(64, 64);
 
-  trainBtn = createButton("train");
+  trainBtn = createButton('train');
   trainBtn.mousePressed(train);
-  addDataBtn = createButton("addData");
+  addDataBtn = createButton('addData');
   addDataBtn.mousePressed(addData);
-  resultLabel = createDiv("I see:");
-
-  labelInput = createInput();
+  resultLabel = createDiv('');
+  labelInput = createInput('label');
 
   const options = {
-    task: "imageClassification",
+    task: 'imageClassification',
     debug: true,
     inputs: [IMAGE_WIDTH, IMAGE_HEIGHT, IMAGE_CHANNELS],
-    layers: [
-      {
-        type: "conv2d",
-        kernelSize: 5,
-        filters: 8,
-        strides: 1,
-        activation: "relu",
-        kernelInitializer: "varianceScaling",
-      },
-      {
-        type: "maxPooling2d",
-        poolSize: [2, 2],
-        strides: [2, 2],
-      },
-      {
-        type: "conv2d",
-        kernelSize: 5,
-        filters: 8,
-        strides: 1,
-        activation: "relu",
-        kernelInitializer: "varianceScaling",
-      },
-      {
-        type: "maxPooling2d",
-        poolSize: [2, 2],
-        strides: [2, 2],
-      },
-      {
-        type: "flatten",
-      },
-      {
-        type: "dense",
-        kernelInitializer: "varianceScaling",
-        activation: "softmax",
-      },
-    ],
   };
 
   // construct the neural network
   nn = ml5.neuralNetwork(options);
-  nn.loadData("daytime_nightime.json", train);
+  // nn.loadData('daytime_nightime.json', train);
```

Comment:
Do we need to keep this comment in?

Reply:
I'll add a comment that explains it further, this is useful in case you don't want to collect your own data and just want to quickly train a model with sample data and see the classification process.